homeassistant/util/uuid.py

```diff
@@ -1,15 +1,12 @@
 """Helpers to generate uuids."""
 
-import random
-import uuid
+from random import getrandbits
 
 
-def uuid_v1mc_hex() -> str:
-    """Generate a uuid1 with a random multicast MAC address.
+def random_uuid_hex() -> str:
+    """Generate a random UUID hex.
 
-    The uuid1 uses a random multicast MAC address instead of the real MAC address
-    of the machine without the overhead of calling the getrandom() system call.
-
-    This is effectively equivalent to PostgreSQL's uuid_generate_v1mc() function
+    This uuid should not be used for cryptographically secure
+    operations.
     """
-    return uuid.uuid1(node=random.getrandbits(48) | (1 << 40)).hex
+    return "%032x" % getrandbits(32 * 4)
```

Comment:
Both these seem really odd if we use this as identifiers for events stored in recorder. Collisions over time seem very likely. We can't we do something much easier like a base number verified to be unique on start, then just incremented by one on each call?

Reply:
I suppose it's a non issue. Given the amount of bits in the random values. So scratch this comment.

Reply:
https://en.wikipedia.org/wiki/Birthday_attack

The probabilities for 128 bits should far exceed the number of events ever fired.